file.go

```diff
@@ -64,6 +67,12 @@ func Empty() *File {
 	return f
 }
 
+func (f *File) reset(name string) {
```

Comment:
What's the use case of this method?

Reply:
Don't know, I think I used it in an early version... I'll remove it.